modules/monitoring-knative-autoscaler-pods.adoc

```diff
@@ -0,0 +1,27 @@
+// Module included in the following assemblies:
+//
+// * serverless/monitoring-serverless.adoc
+
+[id="monitoring-knative-autoscaler-pods_{context}"]
+== Monitoring Knative Serving autoscaler pods using the {product-title} monitoring stack
+
+This section provides example  instructions for the visualization of Knative Serving pod autoscaling metrics by using the {product-title} monitoring tools.
+
+.Prerequisites
+
+* You must have the {product-title} monitoring stack installed. See the documentation about https://docs.openshift.com/container-platform/4.1/monitoring/cluster-monitoring/configuring-the-monitoring-stack.html#configuring-the-monitoring-stack[Configuring the monitoring stack].
+
+.Procedure
+
+. Navigate to the {product-title} Web console and authenticate.
+. Navigate to "Monitoring" → "Metrics".
+. Enter the "Expression" and select "Run queries". To monitor Knative Serving autoscaler pods, use the example expression.
++
+----
+autoscaler_actual_pods
+----
++
+You will now see monitoring information for the Knative Serving autoscaler pods in the console.
+
+.Additional resources
+* For more information about using the monitoring UI, see https://docs.openshift.com/container-platform/4.1/monitoring/cluster-monitoring/prometheus-alertmanager-and-grafana.html#monitoring-accessing-prometheus-alerting-ui-grafana-using-the-web-console_accessing-prometheus[Accessing Prometheus, Alerting UI, and Grafana using the Web console].
```

Comment:
Same note here about the link. If it's to our own docs it should be an xref, which means it can't be included at the module layer.

Reply:
I've updated the files now to just include the modules instead, so lmk if this is OK now once I push the new commit.